Homework6: Ekaterina Zavadskaya

KateZavadskaya opened the pull request:
Add homework6

Add 5 files: hm6_1.py - hm6_5.py

Docs: hm6_1.py - hm6_5.py

KateZavadskaya committed:
Feat(hm6): add homework6

Add 5 files: hm6_1.py - hm6_5.py

Docs: hm6_1.py - hm6_5.py

KateZavadskaya commented:
@Stml89 Can you check my homework6, please.

Stml89 commented on hm6_1.py:
```diff
@@ -0,0 +1,10 @@
+TXT = "Robin Singh"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```
the same issue as in HW5

KateZavadskaya replied:
Understand your message, will fix, thank you.

Stml89 commented on hm6_2.py:
```diff
@@ -0,0 +1,10 @@
+TXT = "I love arrays they are my favorite"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```
here as well

KateZavadskaya replied:
Understand your message, will fix, thank you.

Stml89 reviewed (changes requested):
small comment to fix

KateZavadskaya committed:
Fix(code): add code improvements

Add code improvements:
- hm6_1.py: add variable, delete extra 'split'method
- hm6_2.py: add variable, delete extra 'split'method

Docs: hm6_1.py, hm6_2.py

KateZavadskaya commented:
@Stml89 fixed your comments, check it 🤞, please.
